consensus/istanbul/backend/engine.go

```diff
@@ -643,10 +643,11 @@ func (sb *Backend) updateReplicaStateLoop(bc *ethCore.BlockChain) {
 	}
 }
 
-// SetBlockProcessors implements consensus.Istanbul.SetBlockProcessors
-func (sb *Backend) SetBlockProcessors(hasBadBlock func(common.Hash) bool,
+// SetCallBack implements consensus.Istanbul.SetCallBacks
+func (sb *Backend) SetCallBacks(hasBadBlock func(common.Hash) bool,
```

Comment:
Why this name change?

Reply:
To give it a more appropriate name, since this func injects several call back functions, rather than "block processors".

Reply:
+1 to the change